crosswalks/csdms.csv

```diff
@@ -0,0 +1,74 @@
+Property,csdms
+codeRepository,Source web address
+programmingLanguage,Programming language / Other Programming language
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,module_type
+applicationSubCategory,
+downloadUrl,
+fileSize,
+installUrl,
+memoryRequirements,Memory requirements
+operatingSystem,Supported platforms / Other platform
+permissions,
+processorRequirements,Code optimized for (single/multiple processors)
+releaseNotes,
+softwareHelp,
+softwareRequirements,
+softwareVersion,
+storageRequirements,
+supportingData,
+author, First name / Last name
+citation,
+contributor,
+copyrightHolder,
+copyrightYear,
+dateCreated,Start year development
+dateModified,End year module development
+datePublished,
+editor,
+encoding,
+fileFormat,
+funder,
+keywords,Module domain / model keywords
+license,Program license type
+producer,
+provider,Contact details modeler
+publisher,
+sponsor,
+version,
+isAccessibleForFree,
+isPartOf,Indicate framework if the model is part of a larger framework
+hasPart,
+position,
+description,one-line module description / extended module description
+identifier,DOI name model / For model version / Year version submitted / Link to DOI file
+name,Module name
+sameAs,Also known as
```

Comment:
Not a good match. `sameAs` is for alternative URIs, `Also known as` is for alternative names.